flamedisx/x1t_sr1.py

```diff
@@ -15,21 +19,126 @@
 ##
 # Yield maps
 ##
-
-
 s1_map, s2_map = [
     fd.InterpolatingMap(fd.get_resource(fd.pax_file(x)))
     for x in ('XENON1T_s1_xyz_ly_kr83m-SR1_pax-664_fdc-adcorrtpf.json',
               'XENON1T_s2_xy_ly_SR1_v2.2.json')]
 
+##
+# Parameters
+##
+def_g1 = 0.142
```

Comment:
Default constants are ideally named in all caps, and written out fully, e.g. DEFAULT_G1, DEFAULT_AREA_FRACTION_TOP, etc. 

It is likely a bad idea, but you could consider grouping all of these in a single [immutabledict](https://pypi.org/project/immutabledict/). An ordinary dictionary won't work, then someone will think they can set the entries and then wonder why the function defaults don't change (A: because they are evaluated when the function is defined).